### PR TITLE
[AIR-914] Use distroless image for kube-proxy

### PR DESCRIPTION
**Pull request by joey00072**
### Added security fixes to Kube Proxy

- using distrolessip table images in kube-proxy
- added build script for teamcity

### CODE REVIEW

File: build/pf9-release-image.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# This file will build k8s image. retag it and push image to dockerhub
+make quick-release-images KUBE_EXTRA_WHAT="cmd/kube-proxy" KUBE_BUILD_CONFORMANCE=n KUBE_BUILD_HYPERKUBE=n KUBE_VERBOSE=0 KUBE_BUILD_PLATFORMS=linux/amd64 KUBE_DOCKER_REGISTRY=platform9
```
Comment (cruizen) [inline build/pf9-release-image.sh:4]: Nice to have these flags and build only what's necessary.

**Review by cruizen — APPROVED** [review]

File: build/pf9-release-image.sh (hunk 1 above)
Comment (cruizen) [reply build/pf9-release-image.sh:4]: However, we should add a script / make target later than helps test things locally - including run the conformance tests etc. before releasing an image.

**Comment by cruizen** [comment]
Also talk to @OmkarDeshpande7 about creating a release with a custom `kube-proxy` with security fixes for release-v1.20
(We can take this up later - as a follow-up action)

**Review by Rohitrajak1807 — APPROVED** [review]
Assuming this is tested. All good! Thanks @joey00072

File: build/pf9-release-image.sh (hunk 1 above)
Comment (joey00072) [reply build/pf9-release-image.sh:4]: yeah this script is only for publishing image with teamcity. not intended to use for local development. you can just use `make quick-release-images` for local dev.

**Pull request merged**